ERROR - 2024-11-15 05:59:41 --> mysqli_sql_exception: Table 'cwltechi_dawar.temp_client' doesn't exist in /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php(312): mysqli->query('SELECT *\nFROM `...', 0)
#1 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /home2/cwltechin/public_html/dawar-jewellery/app/Models/HomeModel.php(37): CodeIgniter\Database\BaseBuilder->get()
#5 /home2/cwltechin/public_html/dawar-jewellery/app/Controllers/Home.php(53): App\Models\HomeModel->authenticateUser('123456', '1')
#6 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(933): App\Controllers\Home->authenticateUser()
#7 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /home2/cwltechin/public_html/dawar-jewellery/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-11-15 05:59:41 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'cwltechi_dawar.temp_client' doesn't exist
[Method: POST, Route: authenticateUser]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `temp_client`
WHERE `otp` = :otp:
AND `id` = :id:', [...], false)
 2 APPPATH/Models/HomeModel.php(37): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH/Controllers/Home.php(53): App\Models\HomeModel->authenticateUser('123456', '1')
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->authenticateUser()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 05:59:41 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'cwltechi_dawar.temp_client' doesn't exist
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `temp_client`
WHERE `otp` = \'123456\'
AND `id` = \'1\'')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `temp_client`
WHERE `otp` = \'123456\'
AND `id` = \'1\'')
 3 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `temp_client`
WHERE `otp` = :otp:
AND `id` = :id:', [...], false)
 4 APPPATH/Models/HomeModel.php(37): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH/Controllers/Home.php(53): App\Models\HomeModel->authenticateUser('123456', '1')
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->authenticateUser()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 05:59:41 --> [Caused by] mysqli_sql_exception: Table 'cwltechi_dawar.temp_client' doesn't exist
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('SELECT *
FROM `temp_client`
WHERE `otp` = \'123456\'
AND `id` = \'1\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `temp_client`
WHERE `otp` = \'123456\'
AND `id` = \'1\'')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `temp_client`
WHERE `otp` = \'123456\'
AND `id` = \'1\'')
 4 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `temp_client`
WHERE `otp` = :otp:
AND `id` = :id:', [...], false)
 5 APPPATH/Models/HomeModel.php(37): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH/Controllers/Home.php(53): App\Models\HomeModel->authenticateUser('123456', '1')
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->authenticateUser()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-15 06:02:11 --> mysqli_sql_exception: Table 'cwltechi_dawar.temp_client' doesn't exist in /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php(312): mysqli->query('SELECT *\nFROM `...', 0)
#1 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /home2/cwltechin/public_html/dawar-jewellery/app/Models/HomeModel.php(37): CodeIgniter\Database\BaseBuilder->get()
#5 /home2/cwltechin/public_html/dawar-jewellery/app/Controllers/Home.php(53): App\Models\HomeModel->authenticateUser('123456', '1')
#6 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(933): App\Controllers\Home->authenticateUser()
#7 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /home2/cwltechin/public_html/dawar-jewellery/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-11-15 06:02:11 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'cwltechi_dawar.temp_client' doesn't exist
[Method: POST, Route: authenticateUser]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `temp_client`
WHERE `otp` = :otp:
AND `id` = :id:', [...], false)
 2 APPPATH/Models/HomeModel.php(37): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH/Controllers/Home.php(53): App\Models\HomeModel->authenticateUser('123456', '1')
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->authenticateUser()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 06:02:11 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'cwltechi_dawar.temp_client' doesn't exist
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `temp_client`
WHERE `otp` = \'123456\'
AND `id` = \'1\'')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `temp_client`
WHERE `otp` = \'123456\'
AND `id` = \'1\'')
 3 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `temp_client`
WHERE `otp` = :otp:
AND `id` = :id:', [...], false)
 4 APPPATH/Models/HomeModel.php(37): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH/Controllers/Home.php(53): App\Models\HomeModel->authenticateUser('123456', '1')
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->authenticateUser()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 06:02:11 --> [Caused by] mysqli_sql_exception: Table 'cwltechi_dawar.temp_client' doesn't exist
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('SELECT *
FROM `temp_client`
WHERE `otp` = \'123456\'
AND `id` = \'1\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `temp_client`
WHERE `otp` = \'123456\'
AND `id` = \'1\'')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `temp_client`
WHERE `otp` = \'123456\'
AND `id` = \'1\'')
 4 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `temp_client`
WHERE `otp` = :otp:
AND `id` = :id:', [...], false)
 5 APPPATH/Models/HomeModel.php(37): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH/Controllers/Home.php(53): App\Models\HomeModel->authenticateUser('123456', '1')
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->authenticateUser()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-15 06:22:06 --> mysqli_sql_exception: Table 'cwltechi_dawar.temp_client' doesn't exist in /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php(312): mysqli->query('SELECT *\nFROM `...', 0)
#1 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /home2/cwltechin/public_html/dawar-jewellery/app/Models/HomeModel.php(37): CodeIgniter\Database\BaseBuilder->get()
#5 /home2/cwltechin/public_html/dawar-jewellery/app/Controllers/Home.php(53): App\Models\HomeModel->authenticateTempUser('123457', '1')
#6 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(933): App\Controllers\Home->authenticateUser()
#7 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /home2/cwltechin/public_html/dawar-jewellery/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-11-15 06:22:06 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'cwltechi_dawar.temp_client' doesn't exist
[Method: POST, Route: authenticateUser]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `temp_client`
WHERE `otp` = :otp:
AND `id` = :id:', [...], false)
 2 APPPATH/Models/HomeModel.php(37): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH/Controllers/Home.php(53): App\Models\HomeModel->authenticateTempUser('123457', '1')
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->authenticateUser()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 06:22:06 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'cwltechi_dawar.temp_client' doesn't exist
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `temp_client`
WHERE `otp` = \'123457\'
AND `id` = \'1\'')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `temp_client`
WHERE `otp` = \'123457\'
AND `id` = \'1\'')
 3 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `temp_client`
WHERE `otp` = :otp:
AND `id` = :id:', [...], false)
 4 APPPATH/Models/HomeModel.php(37): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH/Controllers/Home.php(53): App\Models\HomeModel->authenticateTempUser('123457', '1')
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->authenticateUser()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 06:22:06 --> [Caused by] mysqli_sql_exception: Table 'cwltechi_dawar.temp_client' doesn't exist
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('SELECT *
FROM `temp_client`
WHERE `otp` = \'123457\'
AND `id` = \'1\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `temp_client`
WHERE `otp` = \'123457\'
AND `id` = \'1\'')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `temp_client`
WHERE `otp` = \'123457\'
AND `id` = \'1\'')
 4 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `temp_client`
WHERE `otp` = :otp:
AND `id` = :id:', [...], false)
 5 APPPATH/Models/HomeModel.php(37): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH/Controllers/Home.php(53): App\Models\HomeModel->authenticateTempUser('123457', '1')
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->authenticateUser()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 06:39:10 --> ErrorException: Undefined variable $checkUserEmail
[Method: POST, Route: saveWishlist]
in APPPATH/Controllers/Home.php on line 421.
 1 APPPATH/Controllers/Home.php(421): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $checkUserEmail', '/home2/cwltechin/public_html/dawar-jewellery/app/Controllers/Home.php', 421)
 2 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->saveWishlist()
 3 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 06:39:15 --> ErrorException: Undefined variable $checkUserEmail
[Method: POST, Route: saveWishlist]
in APPPATH/Controllers/Home.php on line 421.
 1 APPPATH/Controllers/Home.php(421): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $checkUserEmail', '/home2/cwltechin/public_html/dawar-jewellery/app/Controllers/Home.php', 421)
 2 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->saveWishlist()
 3 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-15 06:47:26 --> mysqli_sql_exception: Unknown column 'gender' in 'field list' in /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `client`...', 0)
#1 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `client`...')
#2 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `client`...')
#3 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client`...', Array, false)
#4 /home2/cwltechin/public_html/dawar-jewellery/app/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update(Array)
#5 /home2/cwltechin/public_html/dawar-jewellery/app/Controllers/Home.php(474): App\Models\HomeModel->updateUser(Array, '')
#6 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(933): App\Controllers\Home->updateUser()
#7 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /home2/cwltechin/public_html/dawar-jewellery/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-11-15 06:47:26 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'gender' in 'field list'
[Method: POST, Route: updateUser]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client` SET `fname` = :fname:, `lname` = :lname:, `gender` = :gender:, `date` = :date:
WHERE `id` = :id:', [...], false)
 2 APPPATH/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update([...])
 3 APPPATH/Controllers/Home.php(474): App\Models\HomeModel->updateUser([...], '')
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->updateUser()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 06:47:26 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'gender' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:47:26\'
WHERE `id` = \'\'')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:47:26\'
WHERE `id` = \'\'')
 3 SYSTEMPATH/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client` SET `fname` = :fname:, `lname` = :lname:, `gender` = :gender:, `date` = :date:
WHERE `id` = :id:', [...], false)
 4 APPPATH/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update([...])
 5 APPPATH/Controllers/Home.php(474): App\Models\HomeModel->updateUser([...], '')
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->updateUser()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 06:47:26 --> [Caused by] mysqli_sql_exception: Unknown column 'gender' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:47:26\'
WHERE `id` = \'\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:47:26\'
WHERE `id` = \'\'')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:47:26\'
WHERE `id` = \'\'')
 4 SYSTEMPATH/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client` SET `fname` = :fname:, `lname` = :lname:, `gender` = :gender:, `date` = :date:
WHERE `id` = :id:', [...], false)
 5 APPPATH/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update([...])
 6 APPPATH/Controllers/Home.php(474): App\Models\HomeModel->updateUser([...], '')
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->updateUser()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-15 06:47:29 --> mysqli_sql_exception: Unknown column 'gender' in 'field list' in /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `client`...', 0)
#1 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `client`...')
#2 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `client`...')
#3 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client`...', Array, false)
#4 /home2/cwltechin/public_html/dawar-jewellery/app/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update(Array)
#5 /home2/cwltechin/public_html/dawar-jewellery/app/Controllers/Home.php(474): App\Models\HomeModel->updateUser(Array, '')
#6 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(933): App\Controllers\Home->updateUser()
#7 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /home2/cwltechin/public_html/dawar-jewellery/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-11-15 06:47:29 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'gender' in 'field list'
[Method: POST, Route: updateUser]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client` SET `fname` = :fname:, `lname` = :lname:, `gender` = :gender:, `date` = :date:
WHERE `id` = :id:', [...], false)
 2 APPPATH/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update([...])
 3 APPPATH/Controllers/Home.php(474): App\Models\HomeModel->updateUser([...], '')
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->updateUser()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 06:47:29 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'gender' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:47:29\'
WHERE `id` = \'\'')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:47:29\'
WHERE `id` = \'\'')
 3 SYSTEMPATH/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client` SET `fname` = :fname:, `lname` = :lname:, `gender` = :gender:, `date` = :date:
WHERE `id` = :id:', [...], false)
 4 APPPATH/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update([...])
 5 APPPATH/Controllers/Home.php(474): App\Models\HomeModel->updateUser([...], '')
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->updateUser()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 06:47:29 --> [Caused by] mysqli_sql_exception: Unknown column 'gender' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:47:29\'
WHERE `id` = \'\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:47:29\'
WHERE `id` = \'\'')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:47:29\'
WHERE `id` = \'\'')
 4 SYSTEMPATH/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client` SET `fname` = :fname:, `lname` = :lname:, `gender` = :gender:, `date` = :date:
WHERE `id` = :id:', [...], false)
 5 APPPATH/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update([...])
 6 APPPATH/Controllers/Home.php(474): App\Models\HomeModel->updateUser([...], '')
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->updateUser()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-15 06:48:02 --> mysqli_sql_exception: Unknown column 'gender' in 'field list' in /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `client`...', 0)
#1 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `client`...')
#2 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `client`...')
#3 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client`...', Array, false)
#4 /home2/cwltechin/public_html/dawar-jewellery/app/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update(Array)
#5 /home2/cwltechin/public_html/dawar-jewellery/app/Controllers/Home.php(474): App\Models\HomeModel->updateUser(Array, '')
#6 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(933): App\Controllers\Home->updateUser()
#7 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /home2/cwltechin/public_html/dawar-jewellery/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-11-15 06:48:02 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'gender' in 'field list'
[Method: POST, Route: updateUser]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client` SET `fname` = :fname:, `lname` = :lname:, `gender` = :gender:, `date` = :date:
WHERE `id` = :id:', [...], false)
 2 APPPATH/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update([...])
 3 APPPATH/Controllers/Home.php(474): App\Models\HomeModel->updateUser([...], '')
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->updateUser()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 06:48:02 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'gender' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:02\'
WHERE `id` = \'\'')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:02\'
WHERE `id` = \'\'')
 3 SYSTEMPATH/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client` SET `fname` = :fname:, `lname` = :lname:, `gender` = :gender:, `date` = :date:
WHERE `id` = :id:', [...], false)
 4 APPPATH/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update([...])
 5 APPPATH/Controllers/Home.php(474): App\Models\HomeModel->updateUser([...], '')
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->updateUser()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 06:48:02 --> [Caused by] mysqli_sql_exception: Unknown column 'gender' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:02\'
WHERE `id` = \'\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:02\'
WHERE `id` = \'\'')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:02\'
WHERE `id` = \'\'')
 4 SYSTEMPATH/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client` SET `fname` = :fname:, `lname` = :lname:, `gender` = :gender:, `date` = :date:
WHERE `id` = :id:', [...], false)
 5 APPPATH/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update([...])
 6 APPPATH/Controllers/Home.php(474): App\Models\HomeModel->updateUser([...], '')
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->updateUser()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-15 06:48:04 --> mysqli_sql_exception: Unknown column 'gender' in 'field list' in /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `client`...', 0)
#1 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `client`...')
#2 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `client`...')
#3 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client`...', Array, false)
#4 /home2/cwltechin/public_html/dawar-jewellery/app/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update(Array)
#5 /home2/cwltechin/public_html/dawar-jewellery/app/Controllers/Home.php(474): App\Models\HomeModel->updateUser(Array, '')
#6 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(933): App\Controllers\Home->updateUser()
#7 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /home2/cwltechin/public_html/dawar-jewellery/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-11-15 06:48:04 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'gender' in 'field list'
[Method: POST, Route: updateUser]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client` SET `fname` = :fname:, `lname` = :lname:, `gender` = :gender:, `date` = :date:
WHERE `id` = :id:', [...], false)
 2 APPPATH/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update([...])
 3 APPPATH/Controllers/Home.php(474): App\Models\HomeModel->updateUser([...], '')
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->updateUser()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 06:48:04 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'gender' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:04\'
WHERE `id` = \'\'')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:04\'
WHERE `id` = \'\'')
 3 SYSTEMPATH/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client` SET `fname` = :fname:, `lname` = :lname:, `gender` = :gender:, `date` = :date:
WHERE `id` = :id:', [...], false)
 4 APPPATH/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update([...])
 5 APPPATH/Controllers/Home.php(474): App\Models\HomeModel->updateUser([...], '')
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->updateUser()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 06:48:04 --> [Caused by] mysqli_sql_exception: Unknown column 'gender' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:04\'
WHERE `id` = \'\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:04\'
WHERE `id` = \'\'')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:04\'
WHERE `id` = \'\'')
 4 SYSTEMPATH/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client` SET `fname` = :fname:, `lname` = :lname:, `gender` = :gender:, `date` = :date:
WHERE `id` = :id:', [...], false)
 5 APPPATH/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update([...])
 6 APPPATH/Controllers/Home.php(474): App\Models\HomeModel->updateUser([...], '')
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->updateUser()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-15 06:48:10 --> mysqli_sql_exception: Unknown column 'gender' in 'field list' in /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `client`...', 0)
#1 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `client`...')
#2 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `client`...')
#3 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client`...', Array, false)
#4 /home2/cwltechin/public_html/dawar-jewellery/app/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update(Array)
#5 /home2/cwltechin/public_html/dawar-jewellery/app/Controllers/Home.php(474): App\Models\HomeModel->updateUser(Array, '')
#6 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(933): App\Controllers\Home->updateUser()
#7 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /home2/cwltechin/public_html/dawar-jewellery/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-11-15 06:48:10 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'gender' in 'field list'
[Method: POST, Route: updateUser]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client` SET `fname` = :fname:, `lname` = :lname:, `gender` = :gender:, `date` = :date:
WHERE `id` = :id:', [...], false)
 2 APPPATH/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update([...])
 3 APPPATH/Controllers/Home.php(474): App\Models\HomeModel->updateUser([...], '')
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->updateUser()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 06:48:10 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'gender' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:10\'
WHERE `id` = \'\'')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:10\'
WHERE `id` = \'\'')
 3 SYSTEMPATH/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client` SET `fname` = :fname:, `lname` = :lname:, `gender` = :gender:, `date` = :date:
WHERE `id` = :id:', [...], false)
 4 APPPATH/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update([...])
 5 APPPATH/Controllers/Home.php(474): App\Models\HomeModel->updateUser([...], '')
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->updateUser()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 06:48:10 --> [Caused by] mysqli_sql_exception: Unknown column 'gender' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:10\'
WHERE `id` = \'\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:10\'
WHERE `id` = \'\'')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:10\'
WHERE `id` = \'\'')
 4 SYSTEMPATH/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client` SET `fname` = :fname:, `lname` = :lname:, `gender` = :gender:, `date` = :date:
WHERE `id` = :id:', [...], false)
 5 APPPATH/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update([...])
 6 APPPATH/Controllers/Home.php(474): App\Models\HomeModel->updateUser([...], '')
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->updateUser()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-15 06:48:45 --> mysqli_sql_exception: Unknown column 'gender' in 'field list' in /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `client`...', 0)
#1 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `client`...')
#2 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `client`...')
#3 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client`...', Array, false)
#4 /home2/cwltechin/public_html/dawar-jewellery/app/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update(Array)
#5 /home2/cwltechin/public_html/dawar-jewellery/app/Controllers/Home.php(474): App\Models\HomeModel->updateUser(Array, '1')
#6 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(933): App\Controllers\Home->updateUser()
#7 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /home2/cwltechin/public_html/dawar-jewellery/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-11-15 06:48:45 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'gender' in 'field list'
[Method: POST, Route: updateUser]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client` SET `fname` = :fname:, `lname` = :lname:, `gender` = :gender:, `date` = :date:
WHERE `id` = :id:', [...], false)
 2 APPPATH/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update([...])
 3 APPPATH/Controllers/Home.php(474): App\Models\HomeModel->updateUser([...], '1')
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->updateUser()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 06:48:45 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'gender' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:45\'
WHERE `id` = \'1\'')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:45\'
WHERE `id` = \'1\'')
 3 SYSTEMPATH/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client` SET `fname` = :fname:, `lname` = :lname:, `gender` = :gender:, `date` = :date:
WHERE `id` = :id:', [...], false)
 4 APPPATH/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update([...])
 5 APPPATH/Controllers/Home.php(474): App\Models\HomeModel->updateUser([...], '1')
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->updateUser()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 06:48:45 --> [Caused by] mysqli_sql_exception: Unknown column 'gender' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:45\'
WHERE `id` = \'1\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:45\'
WHERE `id` = \'1\'')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:45\'
WHERE `id` = \'1\'')
 4 SYSTEMPATH/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client` SET `fname` = :fname:, `lname` = :lname:, `gender` = :gender:, `date` = :date:
WHERE `id` = :id:', [...], false)
 5 APPPATH/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update([...])
 6 APPPATH/Controllers/Home.php(474): App\Models\HomeModel->updateUser([...], '1')
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->updateUser()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-15 06:48:49 --> mysqli_sql_exception: Unknown column 'gender' in 'field list' in /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `client`...', 0)
#1 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `client`...')
#2 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `client`...')
#3 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client`...', Array, false)
#4 /home2/cwltechin/public_html/dawar-jewellery/app/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update(Array)
#5 /home2/cwltechin/public_html/dawar-jewellery/app/Controllers/Home.php(474): App\Models\HomeModel->updateUser(Array, '1')
#6 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(933): App\Controllers\Home->updateUser()
#7 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /home2/cwltechin/public_html/dawar-jewellery/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-11-15 06:48:49 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'gender' in 'field list'
[Method: POST, Route: updateUser]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client` SET `fname` = :fname:, `lname` = :lname:, `gender` = :gender:, `date` = :date:
WHERE `id` = :id:', [...], false)
 2 APPPATH/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update([...])
 3 APPPATH/Controllers/Home.php(474): App\Models\HomeModel->updateUser([...], '1')
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->updateUser()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 06:48:49 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'gender' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:49\'
WHERE `id` = \'1\'')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:49\'
WHERE `id` = \'1\'')
 3 SYSTEMPATH/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client` SET `fname` = :fname:, `lname` = :lname:, `gender` = :gender:, `date` = :date:
WHERE `id` = :id:', [...], false)
 4 APPPATH/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update([...])
 5 APPPATH/Controllers/Home.php(474): App\Models\HomeModel->updateUser([...], '1')
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->updateUser()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 06:48:49 --> [Caused by] mysqli_sql_exception: Unknown column 'gender' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:49\'
WHERE `id` = \'1\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:49\'
WHERE `id` = \'1\'')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `client` SET `fname` = \'Vipul 1\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:49\'
WHERE `id` = \'1\'')
 4 SYSTEMPATH/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client` SET `fname` = :fname:, `lname` = :lname:, `gender` = :gender:, `date` = :date:
WHERE `id` = :id:', [...], false)
 5 APPPATH/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update([...])
 6 APPPATH/Controllers/Home.php(474): App\Models\HomeModel->updateUser([...], '1')
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->updateUser()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-15 06:48:54 --> mysqli_sql_exception: Unknown column 'gender' in 'field list' in /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `client`...', 0)
#1 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `client`...')
#2 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `client`...')
#3 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client`...', Array, false)
#4 /home2/cwltechin/public_html/dawar-jewellery/app/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update(Array)
#5 /home2/cwltechin/public_html/dawar-jewellery/app/Controllers/Home.php(474): App\Models\HomeModel->updateUser(Array, '1')
#6 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(933): App\Controllers\Home->updateUser()
#7 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /home2/cwltechin/public_html/dawar-jewellery/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-11-15 06:48:54 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'gender' in 'field list'
[Method: POST, Route: updateUser]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client` SET `fname` = :fname:, `lname` = :lname:, `gender` = :gender:, `date` = :date:
WHERE `id` = :id:', [...], false)
 2 APPPATH/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update([...])
 3 APPPATH/Controllers/Home.php(474): App\Models\HomeModel->updateUser([...], '1')
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->updateUser()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 06:48:54 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'gender' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `client` SET `fname` = \'Vipul\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:54\'
WHERE `id` = \'1\'')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `client` SET `fname` = \'Vipul\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:54\'
WHERE `id` = \'1\'')
 3 SYSTEMPATH/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client` SET `fname` = :fname:, `lname` = :lname:, `gender` = :gender:, `date` = :date:
WHERE `id` = :id:', [...], false)
 4 APPPATH/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update([...])
 5 APPPATH/Controllers/Home.php(474): App\Models\HomeModel->updateUser([...], '1')
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->updateUser()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 06:48:54 --> [Caused by] mysqli_sql_exception: Unknown column 'gender' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `client` SET `fname` = \'Vipul\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:54\'
WHERE `id` = \'1\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `client` SET `fname` = \'Vipul\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:54\'
WHERE `id` = \'1\'')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `client` SET `fname` = \'Vipul\', `lname` = \'Kumar\', `gender` = \'Male\', `date` = \'2024-11-15 06:48:54\'
WHERE `id` = \'1\'')
 4 SYSTEMPATH/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `client` SET `fname` = :fname:, `lname` = :lname:, `gender` = :gender:, `date` = :date:
WHERE `id` = :id:', [...], false)
 5 APPPATH/Models/HomeModel.php(250): CodeIgniter\Database\BaseBuilder->update([...])
 6 APPPATH/Controllers/Home.php(474): App\Models\HomeModel->updateUser([...], '1')
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->updateUser()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-15 07:14:48 --> mysqli_sql_exception: Table 'cwltechi_dawar.temp_client' doesn't exist in /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `te...', 0)
#1 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `te...')
#2 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `te...')
#3 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `te...', Array, false)
#4 /home2/cwltechin/public_html/dawar-jewellery/app/Models/HomeModel.php(29): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 /home2/cwltechin/public_html/dawar-jewellery/app/Controllers/Home.php(34): App\Models\HomeModel->saveTempUser(Array)
#6 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(933): App\Controllers\Home->saveUser()
#7 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /home2/cwltechin/public_html/dawar-jewellery/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-11-15 07:14:48 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'cwltechi_dawar.temp_client' doesn't exist
[Method: POST, Route: saveUser]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (:fname:, :lname:, :email:, :mobile:, :otp:, :password:, :date:, :status:)', [...], false)
 2 APPPATH/Models/HomeModel.php(29): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH/Controllers/Home.php(34): App\Models\HomeModel->saveTempUser([...])
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->saveUser()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 07:14:48 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'cwltechi_dawar.temp_client' doesn't exist
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (\'aman\', \'singh\', \'aman@gmail.com\', \'1000000001\', \'123456\', \'1234\', \'2024-11-15 07:14:48\', 0)')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (\'aman\', \'singh\', \'aman@gmail.com\', \'1000000001\', \'123456\', \'1234\', \'2024-11-15 07:14:48\', 0)')
 3 SYSTEMPATH/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (:fname:, :lname:, :email:, :mobile:, :otp:, :password:, :date:, :status:)', [...], false)
 4 APPPATH/Models/HomeModel.php(29): CodeIgniter\Database\BaseBuilder->insert([...])
 5 APPPATH/Controllers/Home.php(34): App\Models\HomeModel->saveTempUser([...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->saveUser()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 07:14:48 --> [Caused by] mysqli_sql_exception: Table 'cwltechi_dawar.temp_client' doesn't exist
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (\'aman\', \'singh\', \'aman@gmail.com\', \'1000000001\', \'123456\', \'1234\', \'2024-11-15 07:14:48\', 0)', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (\'aman\', \'singh\', \'aman@gmail.com\', \'1000000001\', \'123456\', \'1234\', \'2024-11-15 07:14:48\', 0)')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (\'aman\', \'singh\', \'aman@gmail.com\', \'1000000001\', \'123456\', \'1234\', \'2024-11-15 07:14:48\', 0)')
 4 SYSTEMPATH/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (:fname:, :lname:, :email:, :mobile:, :otp:, :password:, :date:, :status:)', [...], false)
 5 APPPATH/Models/HomeModel.php(29): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH/Controllers/Home.php(34): App\Models\HomeModel->saveTempUser([...])
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->saveUser()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-15 07:15:29 --> mysqli_sql_exception: Table 'cwltechi_dawar.temp_client' doesn't exist in /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `te...', 0)
#1 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `te...')
#2 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `te...')
#3 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `te...', Array, false)
#4 /home2/cwltechin/public_html/dawar-jewellery/app/Models/HomeModel.php(29): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 /home2/cwltechin/public_html/dawar-jewellery/app/Controllers/Home.php(34): App\Models\HomeModel->saveTempUser(Array)
#6 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(933): App\Controllers\Home->saveUser()
#7 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /home2/cwltechin/public_html/dawar-jewellery/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-11-15 07:15:29 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'cwltechi_dawar.temp_client' doesn't exist
[Method: POST, Route: saveUser]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (:fname:, :lname:, :email:, :mobile:, :otp:, :password:, :date:, :status:)', [...], false)
 2 APPPATH/Models/HomeModel.php(29): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH/Controllers/Home.php(34): App\Models\HomeModel->saveTempUser([...])
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->saveUser()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 07:15:29 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'cwltechi_dawar.temp_client' doesn't exist
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (\'Vipul\', \'Kansal\', \'kansal.vipul612@gmail.com\', \'8800488949\', \'123456\', \'Demo@123456\', \'2024-11-15 07:15:29\', 0)')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (\'Vipul\', \'Kansal\', \'kansal.vipul612@gmail.com\', \'8800488949\', \'123456\', \'Demo@123456\', \'2024-11-15 07:15:29\', 0)')
 3 SYSTEMPATH/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (:fname:, :lname:, :email:, :mobile:, :otp:, :password:, :date:, :status:)', [...], false)
 4 APPPATH/Models/HomeModel.php(29): CodeIgniter\Database\BaseBuilder->insert([...])
 5 APPPATH/Controllers/Home.php(34): App\Models\HomeModel->saveTempUser([...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->saveUser()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 07:15:29 --> [Caused by] mysqli_sql_exception: Table 'cwltechi_dawar.temp_client' doesn't exist
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (\'Vipul\', \'Kansal\', \'kansal.vipul612@gmail.com\', \'8800488949\', \'123456\', \'Demo@123456\', \'2024-11-15 07:15:29\', 0)', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (\'Vipul\', \'Kansal\', \'kansal.vipul612@gmail.com\', \'8800488949\', \'123456\', \'Demo@123456\', \'2024-11-15 07:15:29\', 0)')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (\'Vipul\', \'Kansal\', \'kansal.vipul612@gmail.com\', \'8800488949\', \'123456\', \'Demo@123456\', \'2024-11-15 07:15:29\', 0)')
 4 SYSTEMPATH/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (:fname:, :lname:, :email:, :mobile:, :otp:, :password:, :date:, :status:)', [...], false)
 5 APPPATH/Models/HomeModel.php(29): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH/Controllers/Home.php(34): App\Models\HomeModel->saveTempUser([...])
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->saveUser()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-15 07:16:34 --> mysqli_sql_exception: Table 'cwltechi_dawar.temp_client' doesn't exist in /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `te...', 0)
#1 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `te...')
#2 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `te...')
#3 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `te...', Array, false)
#4 /home2/cwltechin/public_html/dawar-jewellery/app/Models/HomeModel.php(29): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 /home2/cwltechin/public_html/dawar-jewellery/app/Controllers/Home.php(34): App\Models\HomeModel->saveTempUser(Array)
#6 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(933): App\Controllers\Home->saveUser()
#7 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /home2/cwltechin/public_html/dawar-jewellery/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-11-15 07:16:34 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'cwltechi_dawar.temp_client' doesn't exist
[Method: POST, Route: saveUser]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (:fname:, :lname:, :email:, :mobile:, :otp:, :password:, :date:, :status:)', [...], false)
 2 APPPATH/Models/HomeModel.php(29): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH/Controllers/Home.php(34): App\Models\HomeModel->saveTempUser([...])
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->saveUser()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 07:16:34 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'cwltechi_dawar.temp_client' doesn't exist
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (\'Vipul\', \'Kansal\', \'kansal.vipul621@gmail.com\', \'8800488943\', \'123456\', \'Demo@123456\', \'2024-11-15 07:16:34\', 0)')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (\'Vipul\', \'Kansal\', \'kansal.vipul621@gmail.com\', \'8800488943\', \'123456\', \'Demo@123456\', \'2024-11-15 07:16:34\', 0)')
 3 SYSTEMPATH/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (:fname:, :lname:, :email:, :mobile:, :otp:, :password:, :date:, :status:)', [...], false)
 4 APPPATH/Models/HomeModel.php(29): CodeIgniter\Database\BaseBuilder->insert([...])
 5 APPPATH/Controllers/Home.php(34): App\Models\HomeModel->saveTempUser([...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->saveUser()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 07:16:34 --> [Caused by] mysqli_sql_exception: Table 'cwltechi_dawar.temp_client' doesn't exist
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (\'Vipul\', \'Kansal\', \'kansal.vipul621@gmail.com\', \'8800488943\', \'123456\', \'Demo@123456\', \'2024-11-15 07:16:34\', 0)', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (\'Vipul\', \'Kansal\', \'kansal.vipul621@gmail.com\', \'8800488943\', \'123456\', \'Demo@123456\', \'2024-11-15 07:16:34\', 0)')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (\'Vipul\', \'Kansal\', \'kansal.vipul621@gmail.com\', \'8800488943\', \'123456\', \'Demo@123456\', \'2024-11-15 07:16:34\', 0)')
 4 SYSTEMPATH/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (:fname:, :lname:, :email:, :mobile:, :otp:, :password:, :date:, :status:)', [...], false)
 5 APPPATH/Models/HomeModel.php(29): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH/Controllers/Home.php(34): App\Models\HomeModel->saveTempUser([...])
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->saveUser()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-15 13:24:58 --> mysqli_sql_exception: Table 'cwltechi_dawar.temp_client' doesn't exist in /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `te...', 0)
#1 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `te...')
#2 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `te...')
#3 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `te...', Array, false)
#4 /home2/cwltechin/public_html/dawar-jewellery/app/Models/HomeModel.php(29): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 /home2/cwltechin/public_html/dawar-jewellery/app/Controllers/Home.php(34): App\Models\HomeModel->saveTempUser(Array)
#6 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(933): App\Controllers\Home->saveUser()
#7 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /home2/cwltechin/public_html/dawar-jewellery/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-11-15 13:24:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'cwltechi_dawar.temp_client' doesn't exist
[Method: POST, Route: saveUser]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (:fname:, :lname:, :email:, :mobile:, :otp:, :password:, :date:, :status:)', [...], false)
 2 APPPATH/Models/HomeModel.php(29): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH/Controllers/Home.php(34): App\Models\HomeModel->saveTempUser([...])
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->saveUser()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 13:24:58 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'cwltechi_dawar.temp_client' doesn't exist
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (\'Vipul\', \'Kansal\', \'kansal.vipul621@gmail.com\', \'8800488943\', \'123456\', \'Demo@123456\', \'2024-11-15 13:24:58\', 0)')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (\'Vipul\', \'Kansal\', \'kansal.vipul621@gmail.com\', \'8800488943\', \'123456\', \'Demo@123456\', \'2024-11-15 13:24:58\', 0)')
 3 SYSTEMPATH/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (:fname:, :lname:, :email:, :mobile:, :otp:, :password:, :date:, :status:)', [...], false)
 4 APPPATH/Models/HomeModel.php(29): CodeIgniter\Database\BaseBuilder->insert([...])
 5 APPPATH/Controllers/Home.php(34): App\Models\HomeModel->saveTempUser([...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->saveUser()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 13:24:58 --> [Caused by] mysqli_sql_exception: Table 'cwltechi_dawar.temp_client' doesn't exist
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (\'Vipul\', \'Kansal\', \'kansal.vipul621@gmail.com\', \'8800488943\', \'123456\', \'Demo@123456\', \'2024-11-15 13:24:58\', 0)', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (\'Vipul\', \'Kansal\', \'kansal.vipul621@gmail.com\', \'8800488943\', \'123456\', \'Demo@123456\', \'2024-11-15 13:24:58\', 0)')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (\'Vipul\', \'Kansal\', \'kansal.vipul621@gmail.com\', \'8800488943\', \'123456\', \'Demo@123456\', \'2024-11-15 13:24:58\', 0)')
 4 SYSTEMPATH/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (:fname:, :lname:, :email:, :mobile:, :otp:, :password:, :date:, :status:)', [...], false)
 5 APPPATH/Models/HomeModel.php(29): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH/Controllers/Home.php(34): App\Models\HomeModel->saveTempUser([...])
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->saveUser()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-15 13:25:53 --> mysqli_sql_exception: Table 'cwltechi_dawar.temp_client' doesn't exist in /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /home2/cwltechin/public_html/dawar-jewellery/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `te...', 0)
#1 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `te...')
#2 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `te...')
#3 /home2/cwltechin/public_html/dawar-jewellery/system/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `te...', Array, false)
#4 /home2/cwltechin/public_html/dawar-jewellery/app/Models/HomeModel.php(29): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 /home2/cwltechin/public_html/dawar-jewellery/app/Controllers/Home.php(34): App\Models\HomeModel->saveTempUser(Array)
#6 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(933): App\Controllers\Home->saveUser()
#7 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 /home2/cwltechin/public_html/dawar-jewellery/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 /home2/cwltechin/public_html/dawar-jewellery/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /home2/cwltechin/public_html/dawar-jewellery/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-11-15 13:25:53 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'cwltechi_dawar.temp_client' doesn't exist
[Method: POST, Route: saveUser]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (:fname:, :lname:, :email:, :mobile:, :otp:, :password:, :date:, :status:)', [...], false)
 2 APPPATH/Models/HomeModel.php(29): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH/Controllers/Home.php(34): App\Models\HomeModel->saveTempUser([...])
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->saveUser()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 13:25:53 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'cwltechi_dawar.temp_client' doesn't exist
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (\'Vipul\', \'Kansal\', \'kansal.vipul621@gmail.com\', \'8800488943\', \'123456\', \'Demo@123456\', \'2024-11-15 13:25:53\', 0)')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (\'Vipul\', \'Kansal\', \'kansal.vipul621@gmail.com\', \'8800488943\', \'123456\', \'Demo@123456\', \'2024-11-15 13:25:53\', 0)')
 3 SYSTEMPATH/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (:fname:, :lname:, :email:, :mobile:, :otp:, :password:, :date:, :status:)', [...], false)
 4 APPPATH/Models/HomeModel.php(29): CodeIgniter\Database\BaseBuilder->insert([...])
 5 APPPATH/Controllers/Home.php(34): App\Models\HomeModel->saveTempUser([...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->saveUser()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-15 13:25:53 --> [Caused by] mysqli_sql_exception: Table 'cwltechi_dawar.temp_client' doesn't exist
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (\'Vipul\', \'Kansal\', \'kansal.vipul621@gmail.com\', \'8800488943\', \'123456\', \'Demo@123456\', \'2024-11-15 13:25:53\', 0)', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (\'Vipul\', \'Kansal\', \'kansal.vipul621@gmail.com\', \'8800488943\', \'123456\', \'Demo@123456\', \'2024-11-15 13:25:53\', 0)')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (\'Vipul\', \'Kansal\', \'kansal.vipul621@gmail.com\', \'8800488943\', \'123456\', \'Demo@123456\', \'2024-11-15 13:25:53\', 0)')
 4 SYSTEMPATH/Database/BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `temp_client` (`fname`, `lname`, `email`, `mobile`, `otp`, `password`, `date`, `status`) VALUES (:fname:, :lname:, :email:, :mobile:, :otp:, :password:, :date:, :status:)', [...], false)
 5 APPPATH/Models/HomeModel.php(29): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH/Controllers/Home.php(34): App\Models\HomeModel->saveTempUser([...])
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->saveUser()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
